seed/building_sync/mappings.py

```diff
@@ -618,6 +618,16 @@ def update_tree(schema, tree, xpath, target, value, namespaces):
                 'type': 'value',
                 'value': 'text',
                 'formatter': to_float,
+            },
+            'audit_template_building_id': {
+                'xpath': './auc:Buildings/auc:Building/auc:PremisesIdentifiers/auc:PremisesIdentifier[auc:IdentifierCustomName="Audit Template Building ID"]/auc:IdentifierValue',
+                'type': 'value',
+                'value': 'text',
+            },
+            'portfolio_manager_building_id': {
```

Comment:
shouldn't this get mapped to `pm_property_id` since that is the database field name? I'll create a PR to fix this because we need to add these to the default BuildingSync mappings... 

https://github.com/SEED-platform/seed/blob/develop/seed/models/properties.py#L173